decidim-core/lib/decidim/core/engine.rb

```diff
@@ -207,6 +209,145 @@ class Engine < ::Rails::Engine
       initializer "paper_trail" do
         PaperTrail.config.track_associations = false
       end
+
+      initializer "doorkeeper" do
+        Doorkeeper.configure do
+          # Change the ORM that doorkeeper will use (needs plugins)
+          orm :active_record
+
+          # This block will be called to check whether the resource owner is authenticated or not.
+          resource_owner_authenticator do
+            current_user || redirect_to(new_user_session_path)
+          end
+
+          # If you want to restrict access to the web interface for adding oauth authorized applications, you need to declare the block below.
+          # admin_authenticator do
+          #   # Put your admin authentication logic here.
+          #   # Example implementation:
+          #   Admin.find_by_id(session[:admin_id]) || redirect_to(new_admin_session_url)
+          # end
```

Comment:
Should we remove all this commented code?

Reply:
Yes